docs/docs/examples/elasticsearch/BITBUCKET_ELASTICSEARCH.md

```diff
@@ -12,21 +12,31 @@ helm repo add elastic https://helm.elastic.co
 ```
 then install it:
 ```bash
-helm install elasticsearch --version 7.9.3 elastic/elasticsearch
+helm install elasticsearch --set image.tag="7.9.3" elastic/elasticsearch
 ```
+> Running the above commands will install Elasticsearch with default configurations, i.e. 3 worker nodes. 
+> However, it may not always work out of the box if failed to fulfill pre-requisites for the default installation. 
+> Some example pre-requisites include:
+> * cpu/memory requests: 1000m/2Gi
+> * pre-configured storage with at least three 30Gi volumes (one for each worker node)
+> 
+> Refer to [Elasticsearch values.yaml file](https://github.com/elastic/helm-charts/blob/7.9/elasticsearch/values.yaml){.external} for more detail.
```

Comment:
Since moving to mkdocs these callouts `>` should be replaced instead with an [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/)

for example:

```suggestion
!!!warning "Default configuration"
      
      Running the above commands will install Elasticsearch with default configurations, i.e. 3 worker nodes. However, it may not always work out of the box if failed to fulfill pre-requisites for the default installation.

      Pre-requisites include; cpu/memory requests: `1000m/2Gi`, storage with at least three `30Gi` volumes (one for each worker node)
      
      Refer to [Elasticsearch values.yaml file](https://github.com/elastic/helm-charts/blob/7.9/elasticsearch/values.yaml){.external} for more detail.
```

would be rendered as:

![image](https://user-images.githubusercontent.com/409063/131282914-3a8ef8ab-d658-4923-9425-a48919865be4.png)